### ISSUE TITLE
Not Compatible with IE

### ISSUE THREAD

**Issue by HardeepAsrani** [yoarts/flat#91]
This theme is not compatible with recent versions of IE. It's mess in those browsers, more info: https://themeisle.com/forums/topic/internet-explorer/

@ineagu

**Comment by SamPlusPlus** [comment]
@HardeepAsrani as you requested here is a screenshot of one of the IE issues.
![flatieissue](https://cloud.githubusercontent.com/assets/740393/7128292/0bcead00-e21e-11e4-9505-753b080cc076.jpg)

You can see on the main page when listing blog entries, the side bar is not carried all the way down, and the footer is messed up. I believe this is the same issue as #87

**Comment by ineagu** [comment]
@HardeepAsrani @SamPlusPlus thanks a lot for reporting, @rodica-andronache can you please take a look ?

Also @HardeepAsrani please note that : https://github.com/Codeinwp/flatx is the correct repository where we changed the structure to the one we are normally use.

**Comment by SamPlusPlus** [comment]
Was this issue resolved? I didn't see a pull request with it.

**Comment by ineagu** [comment]
@SamPlusPlus as I have said below it was fixed in our version of the repo : https://github.com/Codeinwp/FlatX/commits/development .

@rodica-andronache can you upload it also on wp.org ?

**Comment by SamPlusPlus** [comment]
I see, I was looking at your repro, but I didn't switch branches to development. I apologize. Do you know if this is the same issue as https://github.com/Codeinwp/flat/issues/87?

**Comment by rodica-andronache** [comment]
@ineagu I uploaded the new version. @SamPlusPlus the update should also fix #87, yes.

**Comment by SamPlusPlus** [comment]
@rodica-andronache  thanks for uploading this, however my wordpress installation does not say there is a new theme available. Wordpress says it was lat updated March 29, 2015?

https://wordpress.org/themes/flat/

**Comment by rodica-andronache** [comment]
@SamPlusPlus they are still reviewing the theme, is not been approved yet.

**Comment by k4kfh** [comment]
As far as I know, the footer issue is taken care of. I'm using IE11 on Windows 7 and my footer worked fine, even after I edited the text and styling a bit. Stays at the bottom of the page on every page I tested on a client site, so I think we can mark this off as taken care of.
Used snipping tool just to show:
![image](https://cloud.githubusercontent.com/assets/10699911/7446736/d5419b78-f1a7-11e4-9d47-575af2a5e920.png)
